packages/k/keysmith/package.yml

```diff
@@ -0,0 +1,27 @@
+name       : keysmith
+version    : 23.08.4
+release    : 1
+source     :
+    - https://download.kde.org/stable/release-service/23.08.4/src/keysmith-23.08.4.tar.xz : 76100bf9a038d8bbfdcfe434c02c8362f654c70051b2c1e13055ff7b2031d585
+homepage   : https://apps.kde.org/keysmith
+license    : GPL-3.0-or-later
+component  : desktop.kde
+summary    : OTP client for Plasma Mobile and Desktop
+description: |
+    Keysmith is an application to generate two-factor authentication (2FA) tokens when logging in to your (online) accounts. Currently it supports both HOTP and TOTP tokens.
+builddeps  :
+    - pkgconfig(libsodium)
+    - pkgconfig(Qt5Core)
```

Comment:
Capitalized pkgconfig deps are ordered above lowercase ones, so the three `Qt5*` deps should be above the `libsodium` one.